_posts/en/newsletters/2025-01-10-newsletter.md

```diff
@@ -0,0 +1,182 @@
+---
+title: 'Bitcoin Optech Newsletter #336'
+permalink: /en/newsletters/2025/01/10/
+name: 2025-01-10-newsletter
+slug: 2025-01-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Investigating mining pool behavior before fixing a Bitcoin Core bug:**
+  Abubakar Sadiq Ismail [posted][ismail double] to Delving Bitcoin about
+  a [bug][bitcoin core #21950] discovered in 2021 by Antoine Riard that
+  results in nodes reserving 2,000 vbytes in block templates for coinbase
+  transactions rather than the intended 1,000 vbytes.  Each template
+  could include approximately five additional small transactions if the
+  double reservation was eliminated.  However, that could lead to miners
```

Comment:
nit:
The number 5 is relative to the size of the small transactions, maybe indicate?

Reply:
Leaving as-is.  Anyone who cares can divide 1,000 vbytes by five 5 transactions to figure out that we're talking about roughly 200-vbyte transactions.  We add this detail to help people quickly understand the impact of this bug, which is (at worst) five 2-in-2-out transactions; providing them additional information that they can easily figure out for themselves if they care makes it less quick and less useful, IMO.